### PR TITLE
release version 0.0.61 of the mta plugin.

### PR DESCRIPTION
**Pull request by johnsteele**
Signed-off-by: John Steele <programjsteele@gmail.com>

### CODE REVIEW

File: che-theia-plugins.yaml [hunk 1]
```diff
@@ -322,7 +322,7 @@ plugins:
           attributes:
             protocol: http
     extensions:
-      - https://download.jboss.org/jbosstools/adapters/stable/mta-vscode-extension/mta-vscode-extension-0.0.56-769.vsix
+      - https://download.jboss.org/jbosstools/adapters/stable/mta-vscode-extension/mta-vscode-extension-0.0.61-39.vsix
```
Comment (ericwill) [inline che-theia-plugins.yaml:325]: This link 404's

File: che-theia-plugins.yaml [hunk 2]
```diff
@@ -309,7 +309,7 @@ plugins:
       mountSources: true
       volumeMounts:
         - name: mta-vscode 
-          path: "/home/theia/home/theia/redhat.rhamt-vscode-extension"
+          path: "/home/theia/mta/redhat.mta-vscode-extension"
```
Comment (ericwill) [inline che-theia-plugins.yaml:312]: Is this change intentional? AFAIK the `/home/theia/home/theia` path is a workaround for plugins that hardcode such a path. Has that been fixed in your extension?

**Review by ericwill — CHANGES_REQUESTED** [review]

File: che-theia-plugins.yaml (hunk 2 above)
Comment (johnsteele) [reply che-theia-plugins.yaml:312]: @ericwill Yes, I have updated the path within the extension to use '/home/theia/mta/redhat.mta-vscode-extension'

**Review by ericwill — CHANGES_REQUESTED** [review]
the `revision` field also needs to be updated to the tag/branch/SHA1 in the repository that matches the version of the vsix file linked.

In this case it would be: https://github.com/windup/rhamt-vscode-extension/releases/tag/0.0.61

**Comment by johnsteele** [comment]
@ericwill I've updated the version field.

**Review by ericwill — APPROVED** [review]
